packages/app/analytics/constants.ts

```diff
@@ -0,0 +1,7 @@
+export type EventName = string;
+
+export const EVENTS = {
+  CLICK: {
+    STRATEGY_CARD: "click/strategy-card",
```

Comment:
Wondering if with the new `cardClickEventName` function, we still need this constant file. 

Because is just being used there. I think we can add directly there.

What do you think?

Reply:
I did it this way just in case the future we create more click events or other event categories (like internal/external link events to our social platforms or form submissions that are not clicked)

Reply:
Okay we can keep it for now.